src/hist.rs

```diff
@@ -0,0 +1,103 @@
+extern crate telemetry;
+
+use self::telemetry::plain::*;
+use std::collections::HashMap;
+
+pub struct Histograms {
```

Comment:
Does this need to be public? Is it public for testing purposes? If only for testing, consider adding "test-only" getter methods on Histograms that retrieve information from the Histograms private data.